### PR TITLE
Fixing travis

### PR DESCRIPTION
**Pull request by iparask**

### CODE REVIEW

**Comment by iparask** [comment]
Now it's ready!

**Comment by andre-merzky** [comment]
hehe :-)

File: .travis.yml [hunk 1]
```diff
@@ -19,12 +19,12 @@ env:
     - DIFF=`git diff --name-only --diff-filter=b $OLD...$NEW`
     - DIFF=$(echo $DIFF | grep -o -e '\b[^ ]*.py\b')
     - CMD_PYTEST="coverage run -m pytest -vvv tests/test_component/test_amgr.py;coverage run -m pytest -vvv tests/test_component/test_modules.py;coverage run -m pytest -vvv tests/test_component/test_pipeline.py;coverage run -m pytest -vvv tests/test_component/test_rmgr_2.py;coverage run -m pytest -vvv tests/test_component/test_rmgr.py;coverage run -m pytest -vvv tests/test_component/test_stage.py;coverage run -m pytest -vvv tests/test_component/test_task.py;coverage run -m pytest -vvv tests/test_component/test_tmgr_2.py;coverage run -m pytest -vvv tests/test_component/test_tproc_rp_2.py;coverage run -m pytest -vvv tests/test_component/test_tproc_rp.py;coverage run -m pytest -vvv tests/test_component/test_tmgr.py;coverage run -m pytest -vvv tests/test_component/test_wfp.py;coverage run -m pytest -vvv tests/test_component/test_states.py;coverage run -m pytest -vvv tests/test_integration/test_*;coverage run -m pytest -vvv tests/test_issues/test_*;coverage run -m pytest -vvv tests/test_utils/test_*"
```
Comment (andre-merzky) [inline .travis.yml]: Its not part of the PR, but whats up with this ugly long command line?  could that be done as
```
CMD_PYTEST="for f in tests/test_component/*.py; do coverage run -m pytest -vvv $f; done"
```
or even
```
CMD_PYTEST="coverage run -m pytest -vvv tests/test_component/*.py"
```
?

**Comment by iparask** [comment]
I think it can be merged when travis ends. And then move it to the rest of the branches!

File: .travis.yml (hunk 1 above)
Comment (iparask) [reply .travis.yml]: Yes, you are right. I fixed it
